curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-recursion-by-building-a-decimal-to-binary-converter/64007367d54d2a7efbf44fcf.md

```diff
@@ -19,12 +19,20 @@ isNaN("3.5"); // false
 
 Update the second condition in your `if` statement to use the `isNaN()` function to check if the value returned by `parseInt()` is `NaN`.
 
+Then, add another condition to check if the value returned by `parseInt()` is less then `0`.  This will make sure only positive numbers are accepted by the `checkUserInput` function.
```

Comment:
```suggestion
Then, add another condition to check if the value returned by `parseInt()` is less than `0`.  This will make sure only positive numbers are accepted by the `checkUserInput` function.
```

Reply:
@jdwilkin4, am i wrong jess regarding the grammartical things ?